.github/workflows/build.yml

```diff
@@ -312,45 +312,62 @@ jobs:
       - name: Move coverage files
         if: (success() || failure()) && github.event.pull_request.draft != true && matrix.coverage
         run: |
-          mkdir ./coverage-reports
+          mkdir -p ./coverage/reports
           for shard in ${{ matrix.shards }};
           do
-            mv ./coverage/shard/$shard/lcov.info ./coverage-reports/$shard.txt
```

Comment:
you probably need to leave lcov inside the name

see here for default fileMatch of codecov uploader

https://github.com/codecov/uploader/blob/main/src/helpers/files.ts#L161

Reply:
🤦‍♂️